inc/Integrations/SalesforceB2C/SalesforceB2CIntegration.php

```diff
@@ -18,11 +18,16 @@ public static function register_blocks(): void {
 
 		foreach ( $data_source_configs as $config ) {
 			$data_source = SalesforceB2CDataSource::from_array( $config );
-	
+
 			if ( false === ( $config['service_config']['enable_blocks'] ?? true ) ) {
 				continue;
 			}
-			
+
+			// ToDo: Remove this once existing SFCC data sources have been migrated.
+			if ( empty( $config['service_config']['site_id'] ) ) {
```

Comment:
This is present to stop existing test sites from breaking. It adds the previous value as a default value. Once these test sites have been migrated, this can be removed.

I've added this as an interim fix to add the `site_id` in place with the intention that the internal plan to add migration as a feature would prevent this down the line.